src/main/drivers/vtx_soft_spi_rtc6705.c

```diff
@@ -123,7 +123,21 @@ void rtc6705_soft_spi_set_channel(uint16_t channel_freq)
 
 void rtc6705_soft_spi_set_rf_power(uint8_t reduce_power)
 {
-    rtc6705_write_register(7, (reduce_power ? (PA_CONTROL_DEFAULT | PD_Q5G_MASK) & (~(PA5G_PW_MASK | PA5G_BS_MASK)) : PA_CONTROL_DEFAULT));
+    switch(reduce_power)
+    {
+        case 0:// 1dBm
+            rtc6705_write_register(7, 0x006A);
```

Comment:
For register 7 the bit group I was changing relates to MAI_5G (5G Pre-driver output control). Higher numbers decrease the mW power and lower increases the power. Leaving the rest of the bits default except for min power. 

**The data sheet in this case is not the most helpful at times and any changes to these registers should require real testing with a power meter since you are modifying an active transmitter.**

If you have a power meter tool please test my work for yourself.

I can take pictures of the results if there is more proof needed. My main goal in bringing this up is to reduce issues with people on the field who would think they are in LOW POWER mode when the are actually at full power.

Also note that these power levels will not represent PIT, 25mW, 200mW, 400mW because you are changing the preamp. Instead it will be more like Lowest, Low, Medium, and High depending on the VTX circuit.